types/options_linux.go

```diff
@@ -12,3 +20,33 @@ const (
 var (
 	defaultOverrideConfigFile = "/etc/containers/storage.conf"
 )
+
+// canUseRootlessOverlay returns true if the overlay driver can be used for rootless containers
+func canUseRootlessOverlay(home, runhome string) bool {
+	// we check first for fuse-overlayfs since it is cheaper.
+	if path, _ := exec.LookPath("fuse-overlayfs"); path != "" {
+		//return true
```

Comment:
Why is this commented out?

Reply:
thanks for spotting it, it is a debugging artifact